compiler/z/codegen/OpMemToMem.cpp

```diff
@@ -450,7 +450,7 @@ MemToMemConstLenMacroOp::generateRemainder()
    uint64_t len = (uint64_t)_length;
    TR::Instruction * cursor = (_cursor == NULL ? _cg->getAppendInstruction() : _cursor);
 
-   if ((len >= (uint64_t)(_cg->getS390Linkage())->getLengthStartForSSInstruction()) && len > 0)
+   if (len >= 1)
```

Comment:
Maybe set this into an appropriately named const uint64_t rather than leaving as the almost as generic-as-you-get magic constant '1' ?

Reply:
I agree. Fixed in e46cb2d